README.md

```diff
@@ -1 +1,69 @@
-# budget_app
\ No newline at end of file
+# Budget app
+
+Budget app is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## Built With
+
+- Ruby -v 3.0.2
+- Rails -v 7.0.4
+- gem 'rspec'
+- gem 'sass-rails'
+
+### Presentation
+
+[Video](https://www.loom.com/share/711cc33bc1644273b2a30b881b96c54d)
```

Comment:
- [ ] Good job with the project presentation, however, you missed some steps in your video presentation. Your video presentation only showed the functional aspect of your app and failed to highlight any portion of your code. Kindly ensure that the video contains a highlight about a piece of the code that you find interesting or that you were proud of making.

Reply:
I really apologize for the video. I want to show the code but I changed the operating system soon to ubuntu and I can not share the entire screen that is the reason. I couldn't show the code.